content/enterprise_influxdb/v1.10/tools/api.md

```diff
@@ -212,24 +212,26 @@ The [`/api/v2/delete`](/influxdb/latest/api/#tag/Delete) endpoint accepts `POST`
 Delete all points in a specified time range:
 
 ```bash
-curl --request POST "http://localhost:8086/api/v2/delete/exampleDB/autogen \
+curl --request POST "http://localhost:8086/api/v2/delete?bucket=exampleDB/autogen \
   --header 'Authorization: Token YOUR_API_TOKEN' \
   --header 'Content-Type: application/json' \
   --data '{
     "start": "2020-03-01T00:00:00Z",
     "stop": "2020-11-14T00:00:00Z"
+    }'
 ```
 
 Delete points in a specific measurement with a specific tag value:
 
 ```bash
-curl --request POST "http://localhost:8086/api/v2/delete/exampleDB/autogen \
+curl --request POST "http://localhost:8086/api/v2/delete?bucket=exampleDB/autogen \
   --header 'Authorization: Token YOUR_API_TOKEN' \
   --header 'Content-Type: application/json' \
   --data '{
     "start": "2020-03-01T00:00:00Z",
-    "stop": "2020-11-14T00:00:00Z"
+    "stop": "2020-11-14T00:00:00Z",
     "predicate": "_measurement=\"example-measurement\" AND exampleTag=\"exampleTagValue\""
+    }'
```

Comment:
Should `exampleTagValue` and `example-measurement` be in single quotes here?  We should test this/

https://docs.influxdata.com/enterprise_influxdb/v1.10/query_language/explore-data/#common-issues-with-the-where-clause